index.html

```diff
@@ -111,32 +111,82 @@
-  <dt><dfn id="dom-connectiontype-mixed">mixed</dfn></dt>
-  <dd>The user agent is using multiple connection types as the <a>underlying connection technology</a>.</dd>
-  <dt><dfn id="dom-connectiontype-unknown">unknown</dfn></dt>
-  <dd>The user agent has established a network connection, but is unable to determine what is the <a>underlying connection technology</a>.</dd>
-</dl>
+
+  <section data-dfn-for="ConnectionType" data-link-for="ConnectionType">
+  ### Underlying connection technology
+  This section defines the <dfn data-lt="connection type">connection types</dfn>:
+
+  <dl>
+    <dt><dfn id="dom-connectiontype-bluetooth">bluetooth</dfn></dt>
+    <dd>The user agent is using a Bluetooth connection as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn id="dom-connectiontype-cellular">cellular</dfn></dt>
+    <dd>The user agent is using a cellular connection as the <a>underlying connection technology</a> (e.g., EDGE, HSPA, LTE, etc.).</dd>
+    <dt><dfn id="dom-connectiontype-ethernet">ethernet</dfn></dt>
+    <dd>The user agent is using an Ethernet connection as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn id="dom-connectiontype-none">none</dfn></dt>
+    <dd>The user agent will not contact the network when the user follows links or when a script requests a remote page (or knows that such an attempt would fail) - i.e., equivalent to `navigator.onLine === false` in HTML.</dd>
+    <dt><dfn id="dom-connectiontype-wifi">wifi</dfn></dt>
+    <dd>The user agent is using a Wi-Fi connection as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn id="dom-connectiontype-wimax">wimax</dfn></dt>
+    <dd>The user agent is using a WiMAX connection as the underlying connection technology.</dd>
+    <dt><dfn id="dom-connectiontype-other">other</dfn></dt>
+    <dd>The user agent is using a connection type that is not one of enumerated connection types as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn id="dom-connectiontype-mixed">mixed</dfn></dt>
+    <dd>The user agent is using multiple connection types as the <a>underlying connection technology</a>.</dd>
+    <dt><dfn id="dom-connectiontype-unknown">unknown</dfn></dt>
+    <dd>The user agent has established a network connection, but is unable to determine what is the <a>underlying connection technology</a>.</dd>
+  </dl>
+  </section>
+
+  <section data-dfn-for="EffectiveConnectionType" data-link-for="EffectiveConnectionType">
+  ### Effective connection types
+  This section defines the <dfn data-lt="effective connection type">effective connection types</dfn> (ECT):
+
+  <table id="effective-connection-type-table" >
+    <caption><dfn>Table of effective connection types</dfn></caption>
+    <thead>
+      <tr>
+        <th>ECT</th>
+        <th>Minimum RTT (ms)</th>
+        <th>Maximum downlink (Kbps)</th>
+        <th>Explanation</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><dfn>slow2g</dfn></td>
```

Comment:
Probably it does not matter, but we are now using slow-2g everywhere.
s/slow2g/slow-2g/